tests/bluetooth/host/keys/bt_keys_get_addr/src/main.c

```diff
@@ -167,7 +166,6 @@ static void fff_reset_rule_before(const struct ztest_unit_test *test, void *fixt
 		zassume_true(all_startup_checks_executed == true, NULL);
 	}
 
-	ASSERT_FFF_FAKES_LIST(RESET_FAKE);
```

Comment:
so this test doesn't do assert checking any more?

Reply:
That files doesn't do any asserts, so it's not required here.